pinot-core/src/main/java/org/apache/pinot/core/segment/index/loader/IndexLoadingConfig.java

```diff
@@ -60,6 +60,9 @@
   private boolean _isDirectRealtimeOffheapAllocation;
   private boolean _enableSplitCommitEndWithMetadata;
 
+  // constructed from FieldConfig
+  private Map<String, Map<String, String>> _columnsWithProperties;
```

Comment:
Suggest renaming it to `_columnProperties`. Also don't set it inside the `extractTextIndexColumnsFromTableConfig()`, set it in `extractFromTableConfig()` so that other index type can also access it.

Reply:
done